### PR TITLE
Arkane e0

### PR DESCRIPTION
**Pull request by davidfarinajr**
### Motivation or Problem
Arkane incorrectly adds the atom thermal corrections (0->298K) to E0, so the output E0 is incorrect.

### Description of Changes
Separated atom corrections at 0K from the thermal correction.  Add in 0K correction to E0 in statmech, and add the atoms thermal correction to get H298 in Thermojob.

### CODE REVIEW

**Comment by amarkpayne** [comment]
@alongd do you mind taking a quick look at this to make sure that this is the behavior we want from Arkane? If this looks good to you I'll go forward as the reviewer.

@davidfarinajr do you mind going ahead and pushing commits to fix the 1 Codacy issue as well as updating the unit tests that are failing? (assuming that they aren't failing due to an issue with the code)

File: arkane/encorr/corr.py [hunk 1]
```diff
@@ -121,10 +122,12 @@ def get_atom_correction(model_chemistry, atoms, atom_energies=None):
             )
 
     # Step 2: Atom energy corrections to reach gas-phase reference state
-    atom_enthalpy_corrections = {symbol: data.atom_hf[symbol] - data.atom_thermal[symbol] for symbol in data.atom_hf}
+    atom_enthalpy_corrections = {symbol: (
+        data.atom_hf[symbol], data.atom_thermal[symbol]) for symbol in data.atom_hf}
```
Comment (alongd) [inline arkane/encorr/corr.py]: LGTM, @cgrambow, could you take a look at this line and approve as well?

File: arkane/thermo.py [hunk 2]
```diff
@@ -157,6 +157,8 @@ def generate_thermo(self):
         else:
             species.thermo = wilhoit
 
+        species.thermo.E0 = conformer.E0
```
Comment (alongd) [inline arkane/thermo.py]: What is the function of this addition?

File: arkane/statmech.py [hunk 3]
```diff
@@ -483,7 +483,7 @@ def load(self, pdep=False, plot=False):
             else:
                 e_electronic *= constants.E_h * constants.Na  # convert Hartree/particle into J/mol
             if self.applyAtomEnergyCorrections:
-                atom_corrections = get_atom_correction(self.modelChemistry,
+                atom_corrections_0K, atom_corrections_298K  = get_atom_correction(self.modelChemistry,
```
Comment (alongd) [inline arkane/statmech.py]: Minor: please remove one extra space before the `=` assignment

File: arkane/statmech.py [hunk 4]
```diff
@@ -510,6 +510,7 @@ def load(self, pdep=False, plot=False):
             logging.debug('         E0 (0 K) = {0:g} kcal/mol'.format(e0 / 4184.))
 
         conformer.E0 = (e0 * 0.001, "kJ/mol")
+        self.species.props['atom_corrections_298K'] = atom_corrections_298K
```
Comment (alongd) [inline arkane/statmech.py]: from the  tests:
`AttributeError: 'rmgpy.species.TransitionState' object has no attribute 'props'`

File: arkane/encorr/corr.py (hunk 1 above)
Comment (alongd) [reply arkane/encorr/corr.py]: This introduces a large change of 17 kJ/mol for H2O2, from the tests:
```
FAIL: Test specifying absolute file paths of statmech files
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/arkane/statmechTest.py", line 193, in test_specifying_absolute_file_paths
    self.assertAlmostEqual(h2o2.conformer.E0.value_si, -146031.49933673252)
AssertionError: -128840.68059141742 != -146031.49933673252 within 7 places (17190.818745315104 difference)
```

If this was indeed a bug, then it's a big one. Let's be as confident as we can that this fix is the desired behaviour, i.e., we need a test set with trusted H298's to run on master and on this branch.

**Comment by cgrambow** [comment]
As far as I can tell, you made this PR because you would like the Arkane output file to also show enthalpy at zero Kelvin, correct? You are right that `conformer.E0` is not the zero Kelvin enthalpy, but I would argue that `E0` also doesn't usually refer to zero Kelvin enthalpy so the changes in this PR might just make the additional energy separation more confusing. I think the biggest issue is that the `statmech` module in Arkane is not designed well and should probably be refactored.

**Comment by davidfarinajr** [comment]
I am a little confused as to what `conformer.E0` is in Arkane.  I thought it was the 0K enthalpy: ZPE + electronic_energy.  However, when I calculated a bunch of species with the G4 method in Gaussian, I noticed that H298K was close to ATcT values, but H0K was way off (when assuming `conformer.E0` = H0K).  It looks like `conformer.E0` now is ZPE + electronic_energy + atom thermal corrections (0K to 298K).  This PR remove the atom thermal corrections so that `conformer.E0` = H0K = ZPE + electronic energy.  I agree that it needs refactoring and that the way I did it is very clunky.

File: arkane/encorr/corr.py (hunk 1 above)
Comment (davidfarinajr) [reply arkane/encorr/corr.py]: The ATcT Hf0K for H2O2 is -129 kJ/mol (https://atct.anl.gov/Thermochemical%20Data/version%201.122g/species/?species_number=53) 
We get closer to that with this PR, if `conformer.E0` is suppose to be Hf0K

**Pull request closed without merging**